docs/CONTRIBUTING.md

```diff
@@ -273,7 +273,7 @@ manually from
 Simply press "Run workflow" and choose dry-run.
 
 If a release has failed after the tag has been created and pushed, it can be
-retried by again, pressing "Run workflow", changing "Use workflow from" from
+retried by again, pressing "Run workflow," changing "Use workflow from" from
```

Comment:
Punctuation should stay outside of quotations because we're typically referring to literal values as opposed to speech.